stores/hostdb_test.go

```diff
@@ -546,7 +547,7 @@ func TestRecordScan(t *testing.T) {
 	}
 
 	// The host should still have the subnets.
-	if !reflect.DeepEqual(host.Subnets, subnets) {
+	if !reflect.DeepEqual(host.ResolvedAddresses, resolvedAddresses) {
```

Comment:
Don't ignore the `host.Subnets`. They are deprecated but still need to be tested.